lib/LANraragi/Plugin/Hdoujin.pm

```diff
@@ -0,0 +1,114 @@
+package LANraragi::Plugin::Hdoujin;
+
+use strict;
+use warnings;
+
+#Plugins can freely use all Perl packages already installed on the system
+#Try however to restrain yourself to the ones already installed for LRR (see tools/cpanfile) to avoid extra installations by the end-user.
+use Mojo::JSON qw(from_json);
+
+#You can also use the LRR Internal API when fitting.
+use LANraragi::Model::Plugins;
+
+#Meta-information about your plugin.
+sub plugin_info {
+
+    return (
+        #Standard metadata
+        name      => "Hdoujin",
+        namespace => "Hdoujinplugin",
+        author    => "Pao",
+        version   => "0.1",
+        description =>
+"Collects metadata embedded into your archives by the Hdoujin .json files",

```

Comment:
It'd be good to specify in the description that this plugin does **not** support other HDoujinDL export formats, such as the .txts or ComicInfo .xml files.

Reply:
ok, changed